lib/chat_api/customers.ex

```diff
@@ -246,7 +246,7 @@ defmodule ChatApi.Customers do
   @spec create_customer(map()) :: {:ok, Customer.t()} | {:error, Ecto.Changeset.t()}
   def create_customer(attrs \\ %{}) do
     %Customer{}
-    |> Customer.changeset(attrs)
+    |> Customer.changeset(Map.merge(get_default_params(), attrs))
```

Comment:
I noticed that we weren't setting default props, but we do it for a lot of the other create methods. lmk if you think this shouldn't be here.